bigquery_etl/cli/backfill.py

```diff
@@ -223,29 +233,146 @@ def validate(
+            sql_dir, project_id, status
+        )
+
+    total_backfills_count = 0
+
+    for qualified_table_name, entries in backfills_dict.items():
+        entries_count = len(entries)
+
+        total_backfills_count += entries_count
+
+        project, dataset, table = qualified_table_name_matching(qualified_table_name)
+        status_str = f" with {status} status" if status is not None else ""
+        click.echo(
+            f"""{project}.{dataset}.{table} has {entries_count} backfill(s){status_str}:"""
+        )
+
+        for entry in entries:
+            click.echo(str(entry))
+
+    click.echo(f"\nThere are a total of {total_backfills_count} backfill(s).")
+
+
+@backfill.command(
+    help="""Get information on backfill(s) that require processing.
+
+    Examples:
+
+    # Get info for specific table.
+    ./bqetl backfill scheduled moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6
+
+    \b
+    # Get info for all tables.
+    ./bqetl backfill scheduled
+    """,
+)
+@click.argument("qualified_table_name", required=False)
+@sql_dir_option
+@project_id_option("moz-fx-data-shared-prod")
+@click.pass_context
+def scheduled(ctx, qualified_table_name, sql_dir, project_id):
+    """Return list of backfill(s) that require processing."""
+    total_backfills_count = 0
+
+    backfills_to_process_dict = get_backfill_entries_to_process_dict(
         sql_dir, project_id, qualified_table_name
     )
 
-    total_backfills_count = 0
+    for qualified_table_name, entry_to_process in backfills_to_process_dict.items():
+        total_backfills_count += 1
 
-    for backfill_file, entries in backfills.items():
-        if status is not None:
-            entries = [e for e in entries if e.status.value.lower() == status.lower()]
+        click.echo(f"Backfill entry scheduled for {qualified_table_name}:")
 
-        entries_count = len(entries)
+        # For future us: this will probably end up being a write to something machine-readable for automation to pick up
+        click.echo(str(entry_to_process))
 
-        if entries_count:
-            total_backfills_count += entries_count
+    click.echo(
+        f"\nThere are a total of {total_backfills_count} backfill(s) that require processing."
+    )
 
-            project, dataset, table = extract_from_query_path(backfill_file)
 
-            status_str = f" with {status} status" if status is not None else ""
-            click.echo(
-                f"""{project}.{dataset}.{table} has {entries_count} backfill(s){status_str}:"""
+@backfill.command(
+    help="""Process backfill.yaml with Drafting status that has not yet been processed.
+
+    Examples:
+
+    \b
+    # Process backfill entry for specific table
+    ./bqetl backfill process moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6
+
+    \b
+    # Process entries from all backfill.yaml files if table is not specified
+    ./bqetl backfill process
```

Comment:
Another nit so feel free to ignore but I'd personally limit this functionality to processing a single backfill entry at a time. I think for both the manual and automatic scenarios each backfill process should be it's own isolated "process". That's still possible given the implementation so not a big deal but it would simplify things here a little.

Reply:
I have made the change as suggested.